test/test_autograd.py

```diff
@@ -1752,6 +1752,23 @@ def test_backward_twice_retained_graph_without_saved_values(self):
         c.backward(torch.tensor([1, 1, 1], dtype=torch.double), retain_graph=True)
         c.backward(torch.tensor([1, 1, 1], dtype=torch.double))
 
+    def test_backward_create_graph_warns(self):
+        prev = torch.is_warn_always_enabled()
+        torch.set_warn_always(True)
+
+        b = torch.randn(3, requires_grad=True, dtype=torch.double)
+        c = b * b
+        with warnings.catch_warnings(record=True) as ws:
+            c.backward(torch.ones_like(c), retain_graph=True ,create_graph=True)
+        self.assertTrue(any('Using backward() with create_graph=True' in str(w.message) for w in ws))
```

Comment:
You should set `b.grad = None` here, otherwise you will leak memory as mentioned in the warning you're adding haha

Reply:
Whoops :P